app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -519,7 +519,7 @@ def yum_names_for_job_template(action:, search:, versions: nil)
 
           yum_removable = installed_packages.search_for(search).distinct.pluck(:name)
           if yum_removable.empty?
-            fail N_("No available packages found for search term '%s'. Check the host's content view environments and already-installed packages.")
+            fail N_("Cannot remove package(s): This host does not have any installed packages matching the search term '%s'.")
```

Comment:
forgot to add this ;) You have to define what %s is.
```suggestion
            fail N_("Cannot remove package(s): This host does not have any installed packages matching the search term '%s'.") % search
```

Reply:
@chris1984 It seems the wording changes were lost here